### DISCUSSION TITLE
How to create LtiDeepLinkingResponse in LTI 1.3 using .NET Core?

### DISCUSSION THREAD

**Discussion by kartik-scholarlab** [andyfmiller/LtiLibrary#150] [Q&A]
We are trying to implement LTI 1.3 Deep Link, but we cannot get the LtiDeepLinkingData class reference from anywhere - which library do we need to install? Do you have any ideas on how to proceed with LTI 1.3 Deep Linking implementation from .NET Core? Any help is appreciated

**Comment by jeroenhabets** [comment]
Same question as issue #149 so I kindly refer you there.
